nso-nipap/nso-nipap/src/yang/nipap.yang

```diff
@@ -14,6 +14,70 @@ module nipap {
   import tailf-ncs {
     prefix ncs;
   }
+ 
+  grouping prefix-attributes {
+
+    leaf customer_id {
+      description "Customer identifier";
+      type string;
+      }
+
+    leaf description {
+      description "Description of prefix";
+      type string;
+    }
+
+    leaf node {
+      description "Node, e.g. FQDN";
+      type string;
+    }
+
+    leaf order_id {
+      description "Order identifier";
+      type string;
+    }
+  }
+
+  grouping response-grouping {
+    container response {
+      description "CDB Subscriber will write the response here";
+      config false;
+      tailf:cdb-oper {
+        tailf:persistent true;
+        }
+
+      choice response-choice {
```

Comment:
This looks like somewhat of an anti-pattern to me. Better have a leaf for status, it can be an enum or something with "ok", "error" or whatever. Then you have a separate leaf that contains the error message. Right now you don't even have a leaf under the "ok" case so there's nothing there to read which is über weird ;) because you can't go to one single path and get the status, you have to probe around